app/blueprints/fund/templates/fund.html

```diff
@@ -17,39 +29,28 @@ <h1 class="govuk-heading-l">{{pageHeading}}</h1>
             <fieldset class="govuk-fieldset">
                 <form method="POST">
```

Comment:
Please disable HTML5 validation by adding `novalidate`

```suggestion
                <form method="POST" novalidate>
```

<img width="646" alt="image" src="https://github.com/user-attachments/assets/5f00a8e7-7b31-4a29-9733-c6b88a43318d" />

This is generally recommended practice: https://design-system.w3.org/styles/form-validation.html#:~:text=It%20is%20recommended%20to%20avoid,to%20the%20element.